### ISSUE TITLE
Build failed with missing <event2/event.h>

### ISSUE THREAD

**Issue by Yenya** [plainbanana/Redis-Cluster-Fast#5]
Hello,

I am trying to build Redis::Cluster::Fast using `cpanm Redis::Cluster::Fast` on a clean Fedora 37. It fails with the following:

```
Building Redis-Cluster-Fast
gcc -Isrc -Ideps/build/usr/local/include -Isrc -I/usr/lib64/perl5/CORE -DVERSION="0.05" -DXS_VERSION="0.05" -fPIC -Wall -Wextra -Wdeclaration-after-statement -Wc++-compat -c -D_REENTRANT -D_GNU_SOURCE -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fwrapv -fno-strict-aliasing -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -g -o lib/Redis/Cluster/Fast.o lib/Redis/Cluster/Fast.c
In file included from /usr/lib64/perl5/CORE/perl.h:7242,
                 from lib/Redis/Cluster/Fast.xs:7:
/usr/lib64/perl5/CORE/inline.h: In function ‘Perl_cop_file_avn’:
/usr/lib64/perl5/CORE/inline.h:3489:5: warning: ISO C90 forbids mixed declarations and code [-Wdeclaration-after-statement]
 3489 |     const char *file = CopFILE(cop);
      |     ^~~~~
In file included from deps/build/usr/local/include/hiredis_cluster/adapters/libevent.h:35,
                 from lib/Redis/Cluster/Fast.xs:13:
deps/build/usr/local/include/hiredis/adapters/libevent.h: At top level:
deps/build/usr/local/include/hiredis/adapters/libevent.h:33:10: fatal error: event2/event.h: No such file or directory
```

In fact, the bundled libevent is installed in `deps/libevent`, so the header file is `deps/libevent/include/event2/event.h`.

If I try to build the module against system libevent (`yum install libevent-devel`), the build process tries to use `libevent.a` instead of `libevent.so` shared library, and it fails as well.

I think in the bundled libevent case, it should either install libevent to `deps/usr/local/`, or `-Ideps/libevent/include` should be added to the compiler command line here.

Thanks!

-Yenya

**Comment by plainbanana** [comment]
Hello,

I prepared a clean install of Fedora 37 and performed the same operation and indeed the build failed with the error you mentioned.

In my case, the reason was that I did not have enough packages to build libevent. (patch, libtool and openssl were missing. However, I have already modified it to give `--disable-openssl` since the openssl dependency is not needed, and uploaded to cpan)

c.f. https://github.com/libevent/libevent/blob/master/Documentation/Building.md#linux-deb-like-ubuntudebian

On my environment (Fedora 37), I was able to install with `cpanm -n -v Redis::Cluster::Fast`.

Thanks
